lib/theme/app_theme.dart

```diff
@@ -5,17 +5,31 @@ class AppTheme {
   static ThemeData get light => ThemeData(
         fontFamily: FontFamily.neuzeit,
         textTheme: const TextTheme(
-          bodyMedium: TextStyle(
-            color: Colors.white,
-            fontSize: 17,
-            fontWeight: FontWeight.normal,
-          ),
-          labelMedium: TextStyle(
-            color: Colors.white,
-            fontSize: 17,
-            fontWeight: FontWeight.bold,
-          ),
-        ),
+            bodyMedium: TextStyle(
+              color: Colors.white,
+              fontSize: 17,
+              fontWeight: FontWeight.normal,
+            ),
+            labelMedium: TextStyle(
+              color: Colors.white,
+              fontSize: 17,
+              fontWeight: FontWeight.bold,
+            ),
+            labelSmall: TextStyle(
+              color: Colors.white,
+              fontSize: 13,
+              fontWeight: FontWeight.bold,
+            ),
+            titleLarge: TextStyle(
+              color: Colors.white,
+              fontSize: 34,
+              fontWeight: FontWeight.bold,
+            ),
+            titleMedium: TextStyle(
+              color: Colors.white,
+              fontSize: 28,
+              fontWeight: FontWeight.bold,
+            )),
```

Comment:
We keep the font size from large to small. So please move the `titleLarge` and `titleMedium` to the top 🙏